packages/components/src/components/Label/Label.tsx

```diff
@@ -31,7 +25,7 @@ const Label = React.forwardRef<HTMLLabelElement, LabelProps>(
         fontSize="fontSize10"
         fontWeight="fontWeightMedium"
         lineHeight="lineHeight10"
-        marginBottom={marginBottom}
+        marginBottom="space20"
```

Comment:
Shouldn't we the new tokens here?

Reply:
we could but I preferred to avoid a diff that is harder to read